cmd/experimental/kjobctl/pkg/builder/templates/slurm_entrypoint_script.sh.tmpl

```diff
@@ -11,15 +11,15 @@ if [ ! -d "{{.EnvsPath}}/$JOB_CONTAINER_INDEX" ]; then
 	exit 0
 fi
 
-source {{.EnvsPath}}/$JOB_CONTAINER_INDEX/{{.SbatchEnvFilename}}
+SBATCH_JOB_NAME={{.SbatchJobName}}
 
 export $(cat {{.EnvsPath}}/$JOB_CONTAINER_INDEX/{{.SlurmEnvFilename}} | xargs)
 
 {{.UnmaskFilenameFunction}}
 
-input_file=$(unmask_filename "$SBATCH_INPUT")
-output_file=$(unmask_filename "$SBATCH_OUTPUT")
-error_path=$(unmask_filename "$SBATCH_ERROR")
+input_file=$(unmask_filename "{{.SbatchInput}}")
+output_file=$(unmask_filename "{{.SbatchOutput}}")
+error_path=$(unmask_filename "{{.SbatchError}}")
```

Comment:
Is this is the only usage of `unmask_filename` now? If this is the case it can be converted to a go-template func as in https://pkg.go.dev/text/template#FuncMap.

Reply:
Fixed on https://github.com/kubernetes-sigs/kueue/pull/3180.